app/controllers/users_controller.rb

```diff
@@ -45,6 +46,43 @@ def index
     end
   end
 
+  def show
+    user = User.find(params.require(:id))
+
+    render status: :ok, json: user.as_json(
+      include: %w[roles],
+      methods: %w[special_account_competitions],
+      private_attributes: %w[email],
+    )
+  end
+
+  def merge
+    to_be_maintained_user = User.find(params.require(:toBeMaintainedUserId))
+    to_be_anonymized_user = User.find(params.require(:toBeAnonymizedUserId))
```

Comment:
The names (for both the Ruby variables as well as the corresponding params) are a bit lengthy. My first instinct would be to just drop the "to be" prefix, and call them "maintained user" and "anonymized user".
But I'm open to other ideas as well, as long as they're shorther than the current code.

Reply:
Also, "anonymization" has a special meaning in other places in our code (GDPR etc) so is that really what you're going for here? Is that other user indeed being anonymized in the GDPR sense?
If not, you should reconsider the name here as well.

Reply:
Actually when I think again, anonymization is not really necessary here. The reason why I was thinking to anonymize is that users should not continue use their old account. Or in other words somehow lock access to old account. But it is actually not helping, the user can still create a new account. So I'll remove the anonymizing part from here.

Coming to the variable name, what about `maintained_user` and `non_maintained_user`? Or rather simply `from_user` and `to_user` will work?

Reply:
I like `from_user` and `to_user` very much! But I will leave the final decision with you, because you know what works best for WRT